### PR TITLE
Feature/webclient

### PR DESCRIPTION
**Pull request by tamaspolgar**

### CODE REVIEW

File: project/Aliases.scala [hunk 1]
```diff
@@ -3,8 +3,9 @@ import sbt._
 object Aliases {
 
   lazy val aliases =
-    addCommandAlias("controlGpioExample", ";project examples; ;clean ;runMain org.kaloz.pi4j.client.examples.ControlGpioExample") ++
+    addCommandAlias("controlGpioExample", ";project examples ;clean ;runMain org.kaloz.pi4j.client.examples.ControlGpioExample") ++
       addCommandAlias("listenGpioExample", ";project examples ;clean ;runMain org.kaloz.pi4j.client.examples.ListenGpioExample") ++
       addCommandAlias("shutdownGpioExample", ";project examples ;clean ;runMain org.kaloz.pi4j.client.examples.ShutdownGpioExample") ++
-      addCommandAlias("triggerGpioExample", ";project examples ;clean ;runMain org.kaloz.pi4j.client.examples.TriggerGpioExample")
+      addCommandAlias("triggerGpioExample", ";project examples ;clean ;runMain org.kaloz.pi4j.client.examples.TriggerGpioExample") ++
+      addCommandAlias("webApp", ";project web-client ;clean ;runMain org.kaloz.pi4j.client.web.WebApp")
```
Comment (lachatak) [inline project/Aliases.scala]: I believe it is not needed

File: web-client/src/main/resources/www/js/pinController.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+webClient.controller('PinController', function ($scope) {
+    function init() {
+        $scope.pins = [];
+        $scope.debugEnabled = false;
+        $scope.messages = [];
+        for (i = 0; i != 32; i++) {
+            $scope.pins[i] = {id: i, mode: "disabled", value: null};
+        }
+        $scope.webSocket = new WebSocket("ws://localhost:8080/ws-echo");
```
Comment (lachatak) [inline web-client/src/main/resources/www/js/pinController.js]: ws-echo?

File: web-client/src/main/scala/org/kaloz/pi4j/client/web/Configuration.scala [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package org.kaloz.pi4j.client.web
+
+import com.typesafe.config.ConfigFactory
+
+trait Configuration {
+
+  private lazy val config = ConfigFactory.load("pi4j-client-web")
+
+  lazy val port = config.getInt("port")
```
Comment (lachatak) [inline web-client/src/main/scala/org/kaloz/pi4j/client/web/Configuration.scala]: meaningful config name?

File: web-client/src/main/scala/org/kaloz/pi4j/client/web/WebApp.scala [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package org.kaloz.pi4j.client.web
+
+import scala.io.StdIn
+
+//WE wont need this
+object WebApp extends App {
```
Comment (lachatak) [inline web-client/src/main/scala/org/kaloz/pi4j/client/web/WebApp.scala]: it is not needed

Comment (tamaspolgar) [inline web-client/src/main/scala/org/kaloz/pi4j/client/web/WebApp.scala]: deleted

File: web-client/src/main/scala/org/kaloz/pi4j/client/web/Configuration.scala (hunk 3 above)
Comment (tamaspolgar) [inline web-client/src/main/scala/org/kaloz/pi4j/client/web/Configuration.scala]: changed to webClient.port

File: web-client/src/main/resources/www/js/pinController.js (hunk 2 above)
Comment (tamaspolgar) [inline web-client/src/main/resources/www/js/pinController.js]: changed to pi4jWebSocket

File: project/Aliases.scala (hunk 1 above)
Comment (tamaspolgar) [inline project/Aliases.scala]: removed

**Pull request merged**